library/modules/Filesystem.cpp

```diff
@@ -99,7 +99,13 @@ bool Filesystem::mkdir (std::string path)
 {
     int fail;
 #ifdef _WIN32
-    fail = ::_mkdir(path.c_str());
+    ::std::wstring wide_path;
+    int convertResult = MultiByteToWideChar(437, 0, path.c_str(), (int)strlen(path.c_str()), NULL, 0);
```

Comment:
Does this assume the input is CP437? If so, I don't think that's what we want, and I believe that came up in #3465 too. Requiring CP437 input would severely restrict the characters that callers of these functions can use in paths.

Reply:
I may have misread. Thought I was converting *from* CP437 to CP1252, If I mixed those up, then my mistake. Is 437 not the default in DF?

Reply:
CP437 is the default for text displayed on the DF screen. It's very much not the default for text in our code. We have a few CP437 <> UTF8 conversion functions, so I think UTF8 would be a reasonable assumption.